airbyte-integrations/connectors/source-shopify/source_shopify/spec.json

```diff
@@ -81,6 +81,7 @@
         "description": "The date you would like to replicate data from. Format: YYYY-MM-DD. Any data before this date will not be replicated.",
         "examples": ["2021-01-01"],
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$",
+        "format": "date",
```

Comment:
It will break if we expect a different format of date or date-time, but we are only justifying the format here, we do not change it.  So the answer is - no.